superflore/generators/ebuild/overlay_instance.py

```diff
@@ -19,14 +19,14 @@ def get_random_branch_name():
 
 
 class ros_overlay(repo_instance):
```

Comment:
That's a good point.

I think I should also move the `pull_request` function into `RepoInstance`